lib/Config.js

```diff
@@ -469,7 +470,13 @@ class Config {
             if (process.env.S3AUTH_CONFIG) {
                 authfile = process.env.S3AUTH_CONFIG;
             }
-            const authData = require(authfile);
+            let authData = require(authfile);
```

Comment:
I guess we are ok throwing an error if `authfile` is invalid JSON, @LaurenSpiegel can confirm.